context/app/static/js/pages/Genes/Genes.jsx

```diff
@@ -25,7 +29,7 @@ function Genes() {
           Gene
         </Typography>
         <Typography variant="h1" component="h2">
-          CD4
+          {`CD4 (${commonName})`}
```

Comment:
If I'm not mistaken, `CD4` was an example of a gene name/test data and can be removed here:

```suggestion
          {commonName}
```